yksgi/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "yksgi"
+version = "0.1.0"
+authors = ["The Yk Developers"]
+edition = "2018"
```

Comment:
may as well use rust 2021

Reply:
We don't in any of the other cargo files though.